plugins/catalog/src/components/CatalogPage/CatalogPage.tsx

```diff
@@ -59,7 +59,11 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-const CatalogPageContents = () => {
+export type CatalogPageOpts = {
```

Comment:
I think we would usually call things like this `Props`, since by convention that's what the first argument of a React component is called. So this would be `CatalogPageProps`

Reply:
Happy to rename it, I was following this naming from the [catalog-import](https://github.com/backstage/backstage/blob/master/plugins/catalog-import/src/components/ImportComponentPage.tsx#L51) plugin 😄

Reply:
I renamed it 🎉